nomad/var_files/remote_call_rate.vars

```diff
@@ -0,0 +1,3 @@
+scenario-name = "remote_call_rate"
+duration = 300
```

Comment:
None of the other scenarios have a duration specified in the vars. I noticed that scenario runs on Nomad clients end somehow sometime without a duration. But I added one anyway. Tell me your story.